src/Symfony/Bundle/DebugBundle/README.md

```diff
@@ -0,0 +1,13 @@
+DebugBundle
+===========
+
+DebugBundle provides a tight integration of the Symfony VarDumper component into
+the Symfony full-stack framework.
```

Comment:
Worth to mention it provides the integration of the `ServerLogCommand` from `symfony/monolog-bridge` as well?

Reply:
I added it.